### ISSUE TITLE
Release 0.1.1 assets - linux64 binary contains version 0.1.0

### ISSUE THREAD

**Issue by schindlersebastian** [cloudfoundry/stack-auditor#51]
Problem:
The stack-auditor 0.1.1 release linux64 binary asset still contains verison 0.1.0.

Reproduce:
Download https://github.com/cloudfoundry/stack-auditor/releases/download/v0.1.1/stack-auditor-linux-64
Install it via `cf install-plugin stack-auditor-linux-64` outputs:    
```
Attention: Plugins are binaries written by potentially untrusted authors.
Install and use plugins at your own risk.
Do you want to install the plugin stack-auditor-linux-64? [yN]: y
Installing plugin StackAuditor...
OK

Plugin StackAuditor 0.1.0 successfully installed.
```  
Verify the installed version:
```
cf plugins
Listing installed plugins...

plugin         version   command name   command help
StackAuditor   0.1.0     audit-stack    List all apps with their stacks, orgs, and spaces
StackAuditor   0.1.0     change-stack   Change an app's stack in the current space and restart the app
StackAuditor   0.1.0     delete-stack   Delete a stack from the foundation
```

**Comment by evanfarrar** [comment]
Ah, good catch, we originally thought we would overwrite the old binaries (because this was just a re-build without code changes) but ended up creating a new release and forgot we did not change the version number. We will fix version 0.1.1's binaries.

**Comment by MerricdeLauney** [comment]
Thanks for catching. We uploaded new binaries with the correct versions
